data/mods/Aftershock/items/afs_weapons.json

```diff
@@ -152,7 +152,7 @@
     "looks_like": "bionic_general",
     "color": "magenta",
     "skill": "launcher",
-    "ammo": "40mm",
+    "ammo": [ "40x46mm", "40x53mm" ],
```

Comment:
This needs to just be 40x46mm.
40x53mm is only used in mounted weapons, and they are not compatible.

Reply:
Done.